user-manual/transfer/dataverse.rst

```diff
@@ -0,0 +1,118 @@
+.. _dataverse-transfers:
+
+=====================
+Dataverse Integration
+=====================
+
+.. _dataverse-overview:
+
+Overview
+--------
+
+`Dataverse`_ is an open source repository for research data. Archivematica can
+be configured to use a Dataverse Repository as a Transfer Source Location. 
+Dataverse Transfer Source Locations can be configured to display all available 
+datasets or a subset using query parameters.
+
+Datasets are retrieved directly using the Dataverse API and processed like 
+standard transfers, with some additional processing as described below.
+
+Dataverse integration is supported by Archivematica version 1.8 and above, and
+has been developed and tested using Dataverse version 4.8.6
+
+*On this page*
+
+* :ref:`Selecting Datasets for preservation <selecting-datasets>`
+* :ref:`Processing Dataverse datasets <processing-datasets>`
+* :ref:`Dataverse METS file <dataverse-mets>`
+* :ref:`AIP METS file <dataverse-aip-mets>`
+* :ref:`Configuration <dataverse-configuration>`
+
+.. _selecting-datasets:
+
+Selecting Datasets for preservation
+-----------------------------------
+
+When a Dataverse Transfer Source Location is selected in the Transfer Tab of 
+the Dashboard, users can browse a list of available datasets. Selecting a 
+directory icon will expand the view to display the list of files included in 
+the dataset. Individual files can’t be selected for Transfer.
+
+When a Dataverse dataset is selected, the Transfer Type ‘Dataverse’ must also 
+be selected.
+
+.. _processing-datasets:
+
+Processing Dataverse datasets
+-----------------------------
+
+Archivematica creates a Dataverse METS file to describe the contents and 
+structure of the dataset as retrieved from Dataverse. Archivematica also 
+creates an agents.json file, that includes details of the Dataverse instance 
+configured in the Storage Service. This information is used to populate the 
+Dataverse PREMIS agent details in the AIP METS. 
+
+Fixity checks are conducted using any checksums provided by Dataverse. Other 
+microservices are carried out as normal (and as configured in the processing 
+configuration). The final AIP will contain descriptive metadata provided by 
+Dataverse, attributes to indicate any derivatives generated by Dataverse, 
+and attributes to indicate the outcome of fixity checks conducted using 
+checksums provided by Dataverse.
+
+.. _dataverse-mets:
+
+Dataverse METS file
+-------------------
+
+Archivematica generates a Dataverse METS file that describes the contents of 
+the dataset as retrieved from Dataverse. The Dataverse METS includes:
+
+* descriptive metadata about the dataset, mapped to the `DDI standard`_
+
+* a <mets:fileSec> section that lists all files provided, grouped by type 
+  (original, metadata or derivative)
+
+* a <mets:structMap> section that describes the structure of the files as 
```

Comment:
For snippets of xml like this, double inverted quotations are a good idea to make it clearer to read, e.g. ` ``<mets:fileSec>`` `.

![image](https://user-images.githubusercontent.com/1880412/45992984-21b28900-c052-11e8-92c8-7989a49864db.png)

Reply:
thanks for the suggestion. Have updated these with the additional quotations.